ush/python/pygfs/task/land_analysis.py

```diff
@@ -87,12 +87,17 @@ def prepare_GTS(self) -> None:
         logger.info("Copying GTS obs for bufr2ioda.x")
         FileHandler(prep_gts_config.gtsbufr).sync()
 
-        # generate bufr2ioda YAML file
-        bufr2ioda_yaml = os.path.join(self.runtime_config.DATA, "bufr_adpsfc_snow.yaml")
-        logger.info(f"Generate BUFR2IODA YAML file: {bufr2ioda_yaml}")
-        temp_yaml = parse_j2yaml(self.task_config.BUFR2IODAYAML, self.task_config)
-        save_as_yaml(temp_yaml, bufr2ioda_yaml)
-        logger.info(f"Wrote bufr2ioda YAML to: {bufr2ioda_yaml}")
+        # generate bufr2ioda YAML files
+        adpsfc_yaml = os.path.join(self.runtime_config.DATA, "bufr_adpsfc_snow.yaml")
+        logger.info(f"Generate BUFR2IODA YAML file: {adpsfc_yaml}")
+        temp_yaml = parse_j2yaml(self.task_config.BUFRADPSFCYAML, self.task_config)
+        save_as_yaml(temp_yaml, adpsfc_yaml)
+        logger.info(f"Wrote bufr2ioda YAML to: {adpsfc_yaml}")
+        snocvr_yaml = os.path.join(self.runtime_config.DATA, "bufr_snocvr.yaml")
+        logger.info(f"Generate BUFR2IODA YAML file: {snocvr_yaml}")
+        temp_yaml = parse_j2yaml(self.task_config.BUFRSNOCVRYAML, self.task_config)
+        save_as_yaml(temp_yaml, snocvr_yaml)
+        logger.info(f"Wrote bufr2ioda YAML to: {snocvr_yaml}")
```

Comment:
I think we may be able to make this a bit less hard-coded and yaml driven from the top.
Say for e.g. somewhere in GDASApp/parm/land has `bufr2ioda.yaml` that contains:
```yaml
bufr2ioda:
     adpsfc: {{ HOMEgfs }}/sorc/gdas.cd/test/testinput/bufr_adpsfc_snow.yaml
     snocvr: {{ HOMEgfs }}/sorc/gdas.cd/test/testinput/bufr_snocvr.yaml
```
We could then loop over this list to parse the yamls, and call the `bufr2ioda.x` and verify the output is produced.

The code then does not know what it is processing and bufr files can be added/removed to the yaml.

Would this be acceptable?

Reply:
we may be able to use `land/prep/prep_gts.yaml` as it already seems to have the appropriate information and it is likely better to keep them together.

Reply:
@aerorahul Could you please give more clue how to do this? Thanks.

Reply:
I mean how to loop over this list to parse the yamls?

Reply:
@aerorahul Follow your suggestions, I have made the changes for your review. Thanks.